Dangerfile

```diff
@@ -0,0 +1,70 @@
+## Copyright 2022 Google LLC
```

Comment:
Is it possible to move both `Dangerfile` and `Gemfile` under say ci/danger? Having them at the root adds unnecessary noise imho.